.resinci.yml

```diff
@@ -0,0 +1 @@
+disabled: true
```

Comment:
You can remove this file altogether since balenaCI will be auto-disabled if it detects flowzone (it looks for a `.github/workflows/flowzone.yml` file)
```suggestion
```

Reply:
Done @Page- :+1: 
Saw this file in other PRs and thought it was (still) needed.
Any comment on the `engines`?